x-pack/plugins/observability_solution/logs_explorer/common/data_views/models/data_view_descriptor.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DataViewListItem } from '@kbn/data-views-plugin/common';
+import { TIMESTAMP_FIELD } from '../../constants';
+import { DataViewSpecWithId } from '../../dataset_selection';
+import { DataViewDescriptorType } from '../types';
+import { buildIndexPatternRegExp } from '../utils';
+
+type Whitelist = Array<string | RegExp>;
+
+const LOGS_WHITELIST: Whitelist = [
+  buildIndexPatternRegExp(['logs', 'auditbeat', 'filebeat', 'winbeat']),
+  // Add more strings or regex patterns as needed
+];
+
+export class DataViewDescriptor {
+  id: DataViewDescriptorType['id'];
+  dataType: DataViewDescriptorType['dataType'];
+  name: DataViewDescriptorType['name'];
+  namespaces: DataViewDescriptorType['namespaces'];
+  title: DataViewDescriptorType['title'];
+  type: DataViewDescriptorType['type'];
+
+  private constructor(dataViewDescriptor: DataViewDescriptorType) {
+    this.id = dataViewDescriptor.id;
+    this.dataType = dataViewDescriptor.dataType;
+    this.name = dataViewDescriptor.name;
+    this.namespaces = dataViewDescriptor.namespaces;
+    this.title = dataViewDescriptor.title;
+    this.type = dataViewDescriptor.type;
+  }
+
+  getFullTitle() {
+    return this.name;
+  }
+
+  toDataviewSpec(): DataViewSpecWithId {
+    return {
+      id: this.id,
+      name: this.name,
+      timeFieldName: TIMESTAMP_FIELD,
```

Comment:
Sounds correct, this was inherited from the other selections where we create an ad-hoc data view, but since we are switching to a persisted data view in this case, the `timeFieldName`can be omitted and the custom time field will be used 👌